scripts/systems.py

```diff
@@ -219,9 +216,12 @@
 #SBATCH --no-requeue
 
 # Env
-export MPICH_OPT_THREAD_SYNC=0 # Required to work around MPICH bug
+export FI_MR_CACHE_MONITOR=disabled
+export MPICH_GPU_SUPPORT_ENABLED=1
 export MIMALLOC_EAGER_COMMIT_DELAY=0
-export MIMALLOC_LARGE_OS_PAGES=1
+export MIMALLOC_ALLOW_LARGE_OS_PAGES=1
+export DLAF_BT_BAND_TO_TRIDIAG_HH_APPLY_GROUP_SIZE=128
```

Comment:
I know that `gen_dlaf_strong-gpu.py` is just an example, but I just wanted to point out that this variable is also set there directly as an extra flag for the miniapps. Should it be removed from there since there is a risk of divergence?

Reply:
I'm mostly indifferent. Should we put some other "placeholder" flag to replace the group size in the example script just to show what "extra flags" it's referring to?

Reply:
I removed them in https://github.com/eth-cscs/DLA-Future/pull/1209/commits/6f88fe2c0f2b12fb08f85aa1be061f2a23e054c9. Please unresolve if you disagree.